fedimint/ui
# fix: gateway fonts

Closes #139

### What This Does

* Moves font files into `packages/ui/assets` for sharing between gateway and guardian UIs
* Adds font src path props to `<Fonts />` instead of hard-coding a path
  * Both apps now do the `import` themselves to allow their respective bundlers to generate paths, then pass them as props to `<Fonts />`

### Steps to Test

1. Open the guardian UI, confirm both inter and space grotesk fonts load
1. Open the gateway UI, confirm both inter and space grotesk fonts load

## Screenshots

<img width="1007" alt="Screenshot 2023-09-01 at 3 56 10 PM" src="https://github.com/fedimint/ui/assets/649992/d676a0fe-5c6d-45a5-b949-46e8018a1474">
